src/components/forms/controls/DatePicker/DatePicker.module.scss

```diff
@@ -0,0 +1,207 @@
+/* Copyright (c) Fortanix, Inc.
+|* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of
+|* the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@use '../../../../styling/defs.scss' as bk;
+
+// TODO: copied manually from old Baklava, where should we put this?
+$z-index-date-picker: 100;
```

Comment:
Ideally we'd use popover for this. But not sure if we can make that work with this library.

If we start having multiple z-index values again I think we should have a single file for it like we did in previous Baklava.

Reply:
Moved it to variables.scss.